src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-7-0-319.mdx

```diff
@@ -9,6 +9,14 @@ security: []
 ---
 ## New Relic PHP Agent v10.7.0.319
 
+### Important note
+
+If the following error occurs when installing the newrelic-php5 Debian package:
+``` php
+    Setting up newrelic-php5 (10.7.0.319) ...
+    Unknown PHP version: 8.2
+Please run the newrelic-install installation program which will complete the installation of the PHP agent and set up the correct symlink to the PHP extension directory.
```

Comment:
```suggestion
If the following error occurs when installing the newrelic-php5 Debian package:
```c
    Setting up newrelic-php5 (10.7.0.319) ...
    Unknown PHP version: 8.2
Please run the newrelic-install installation program which will complete the installation of the PHP agent and set up the correct symlink to the PHP extension directory.
```